modules/sdk-coin-arbeth/src/arbeth.ts

```diff
@@ -33,4 +33,22 @@ export class Arbeth extends AbstractEthLikeNewCoins {
     const explorerUrl = common.Environments[this.bitgo.getEnv()].arbiscanBaseUrl;
     return await recoveryBlockchainExplorerQuery(query, explorerUrl as string, apiToken);
   }
+
+  /**
+   * Check whether gas limit passed in by user are within our max and min bounds
+   * If they are not set, set them to the defaults
+   * @param {number} userGasLimit user defined gas limit
+   * @returns {number} the gas limit to use for this transaction
+   */
+  setGasLimit(userGasLimit?: number): number {
```

Comment:
Is this method intentionally duplicated or do we plan to abstract it since both chains have similar methods and stuff

Reason being, these seem like common utility methods and don't seem specific to opeth or arbeth

Reply:
Yeah, actually I have a ticket in the epic to abstract it out, but have been duplicated for now as this was needed on priority for some testing